Update verification of external checksums section

ross-spencer opened the pull request:
Users can make use of externally generated checksums during transfer.
Archivematica 1.10 will see a minor change to that step to ensure
that checksums are properly validated. This commit documents that
modification plus lists SHA512 as a new algorithm that can be used.

Connected to archivematica/issues#346 
Connected to archivematica/Issues#796

ross-spencer committed:
Update verification of external checksums section

Users can make use of externally generated checksums during transfer.
Archivematica 1.10 will see a minor change to that step to ensure
that checksums are properly validated. This commit documents that
modification plus lists SHA512 as a new algorithm that can be used.

sromkey commented:
@ross-spencer What issue can we connect this to? Otherwise, looks good!! Thanks!!

Edited: I see it now, sorry :man_facepalming:

sromkey reviewed (approved):
lgtm!!

ross-spencer commented:
That's great! Thanks @sromkey I'll merge these later next week once the code PR is done too. Thank you!

ross-spencer committed:
Remove Blake2 from manual

ross-spencer commented:
Hi @sallain and @sromkey I've added a new commit on top of the previously approved changes and requested re-review. 

Following the issues with Blake2 I've removed it from the docs now, and updated the text in the first commit as well. Fortunately it wasn't merged! (Small mercies!). This should complete the triumvirate of changes needed to remove Blake from `1.10` support right now. 

* Connected to https://github.com/archivematica/Issues/issues/796
